ssl-impl/src/test/java/io/crate/protocols/postgres/SslReqHandlerTest.java

```diff
@@ -120,8 +114,10 @@ public void testClassLoadingFallback() {
         }
     }
 
-    @Test(expected = SslConfigurationException.class)
+    @Test
     public void testClassLoadingWithInvalidConfiguration() {
+        expectedException.expect(SslConfigurationException.class);
+        expectedException.expectMessage("Failed to build SSL configuration");
```

Comment:
In general, grabbing for exception messages should be the last resort because they provide information to the user but IMHO should not be part of the code flow.